### PR TITLE
Refactors autodeploy for new source and drupal.org

### PR DESCRIPTION
**Pull request by typhonius**
Addresses #23

### CODE REVIEW

File: build/acsf/phing/autodeploy/autodeploy.sh [hunk 1]
```diff
@@ -27,11 +25,8 @@ mirror_push() {
...
 
-  # Use git filter-branch to remove ACSF specifics.
-  cd ${DEPLOY}
-  git filter-branch --force --index-filter 'git rm -r --cached --ignore-unmatch acsf build/acsf' --prune-empty --tag-name-filter cat -- --all
-
   # Add our remotes & PUSH.HIM
+  cd ${DEPLOY}
   for key in "${!MIRRORS[@]}"; do
     git remote add "${key}" "${MIRRORS[$key]}"
     git push --tags -f "${key}" "${BRANCH}"
```
Comment (seanhamlin) [inline build/acsf/phing/autodeploy/autodeploy.sh]: can we remove `-f` from all push commands?

Comment (typhonius) [inline build/acsf/phing/autodeploy/autodeploy.sh]: We can!

Comment (seanhamlin) [inline build/acsf/phing/autodeploy/autodeploy.sh]: :smile:

**Comment by seanhamlin** [comment]
Just re-reading the full file, and having trouble understanding where we push the commits (I can see the tags). I think the commands it needs is similar to:

`git push drupalorg 7.x-2.x --follow-tags`

Where the remote drupalorg is added earlier. This is mostly from http://stackoverflow.com/questions/3745135/push-git-commits-tags-simultaneously

Note, we never want to push to the master branch of drupal.org as I deleted that :grinning:

**Comment by typhonius** [comment]
I'll rebase this when we're ready to merge

**Comment by seanhamlin** [comment]
Looks good to me, can this be squashed to one commit now?

**Comment by typhonius** [comment]
#squashed :haircut:

**Comment by seanhamlin** [comment]
Re-ran travisCI to remove false positive. Code looks fine from my perspective. Merging in. We need drupal.org to contain the latest code.

**Pull request merged**